src/api/github/webhooks/index.ts

```diff
@@ -0,0 +1,10 @@
+const { createNodeMiddleware, createProbot } = require("probot");
```

Comment:
Any reason why we're using CommonJS `require/module.exports` here when we're using `import/export` elsewhere? (and am using TS, so generally should always just be able to use `import/export`)

Reply:
> Any reason why we're using CommonJS `require/module.exports` here when we're using `import/export` elsewhere? (and am using TS, so generally should always just be able to use `import/export`)

I just removed the code but yep, you are right.